.circleci/config.yml

```diff
@@ -131,12 +131,12 @@ jobs:
             ~/mbx-ci aws setup
       - restore_cache:
           keys:
-            - v4-yarn-{{ checksum "yarn.lock" }}
-      - run: yarn
+            - v4-npm-{{ checksum "package-lock.json" }}
+      - run: npm install
```

Comment:
Should [`npm ci`](https://docs.npmjs.com/cli/v7/commands/npm-ci) be used instead? This will guarantee consistency with the `package-lock.json`, and will error if there is a mismatch between the `package.json` and `package-lock.json`.